clearmash/pipeline-spec.yaml

```diff
@@ -57,6 +57,10 @@ download-related-entities:
         add-resource: entities
         schema: (datapackage_pipelines_mojp.clearmash.pipeline_funcs:entities_schema)
         load-table: clearmash-entities
+    - run: ..datapackage_pipelines_mojp.common.processors.delay_limit
+      parameters:
+        resource: entities
+        stop_after_minutes: 60
```

Comment:
removed in https://github.com/Beit-Hatfutsot/mojp-dbs-pipelines/pull/85/commits/e3fe084c8737d57edd53f5a2761385f6fe3b2d63